### DISCUSSION TITLE
Campo Días Laborales

### DISCUSSION THREAD

**Discussion by EtzelMV** [icebeam7/UTBBackendApi#11] [Ideas]
**Algo así intentabas hacer?**

![image](https://github.com/user-attachments/assets/b0ee525c-0944-45da-92b5-f03e4cec434d)

**Comment by icebeam7** [comment]
Excelente, asi es :D

**Comment by EtzelMV** [comment]
Por si aún no has tenido tiempo de ver cómo colocar esa opción...

![1](https://github.com/user-attachments/assets/d8c3a09f-632f-4200-ae3f-bf440940f753)

![2](https://github.com/user-attachments/assets/17545a5e-6f0f-4b22-a796-d7670c672f00)

![3](https://github.com/user-attachments/assets/9b0e08ad-a6b7-4855-aee4-0e0ed09ead5b)
